nml/actions/action2var_variables.py

```diff
@@ -240,6 +249,26 @@ def signed_byte_parameter(name, args, pos, info):
     'other_veh_z_offset'  : {'var': 0x62, 'start': 24, 'size': 8, 'param_function':signed_byte_parameter, 'value_function':value_sign_extend},
 }
 
+varact2vars60x_trains = {
```

Comment:
Again, would be helpful to link to the OTTD change:
https://github.com/OpenTTD/OpenTTD/commit/868d84bbfc8d2c1471ef713854dd05ebc872615a